libs/coin-framework/src/currencies/support.ts

```diff
@@ -72,7 +71,7 @@ async function initializeUserSupportedFiats() {
 
 export async function fetchSupportedFiatsTokens(): Promise<string[]> {
   try {
-    const response = await fetch(`${getEnv("LEDGER_COUNTERVALUES_API")}/v2/supported-to`, {
+    const response = await fetch(`${getEnv("LEDGER_COUNTERVALUES_API")}/v3/supported/fiat`, {
```

Comment:
I thought we can't do this move yet because the market page does not support all the extra fiats until moved to /v3/markets API.

But when I tried on a build of this PR I still see the the "short" list in the currency selection menu, so maybe I am missing something ?

Reply:
@jnicoulaud-ledger 
- AFAIK market don't uses this function
- the data of this API is currently crossed with hardcoded data: https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledgerjs/packages/cryptoassets/src/fiats.ts#L23 so it's possible that not all fiat appears yet. I can probably complete the list with the missing fiat data (but also indeed in future we would need a way to have this as an API if we don't want to maintain it statically)

Reply:
Ha OK, that explains. But if my understanding is correct, market inherits the globally set currency so we are limited by what market supports.

For the harcoded data, I guess we could expose it in an API, but it's ISO-4217 standard, there are libraries that provide this metadata: https://www.npmjs.com/search?q=keywords:iso4217

Reply:
@jnicoulaud-ledger https://github.com/LedgerHQ/ledger-live/pull/6798

Reply:
i've merged the additions of fiats so we can test more. i seems to have a subset of fiat that all works on market. do you have a specific fiat that works on the new api but not on the old?

Reply:
Yes, it's the diff between https://countervalues.live.ledger.com/v2/supported-to and https://countervalues.live.ledger.com/v3/supported/fiat, so for instance: `AFN`, `ALL`, `AMD`...

Reply:
ok, after testing, i can confirm that market uses a totally different list , so we are fine with these changes. however, I had to do extra fix to actually make Live instance support the new list.

note for ourself: coin-framework also hardcode a `locallySupportedFiats` list, which i suspect market uses, so i'm not touching this, but in future we will need to move this out of coin-framework. ( cc @sprohaszka-ledger fyi )

Reply:
I just tested again:
 - On prod build global currency and market are always synced
 - On this PR, market currency is always USD even if I set a global currency that is supported (EUR)

I would at least expect market currency to follow the global one when possible.

But also I remember last time this was discussed, it was considered not acceptable to have a different set of supported currencies globally and for market, so we might want to double check this ?

Reply:
> I would at least expect market currency to follow the global one when possible.

it doesn't seem it's the case today, and this PR didn't have the scope to address this. therefore if things are not connected i'm not sure if it's going to be a problem to have different list on both sides. indeed it's not ideal but i'm not seeing it a blocker to do this first step.
I'll ask product on monday.

Reply:
seen with Anthony, we're good to have this evolution even if market is disconnected for the time being